### PR TITLE
More ISSUE_TEMPLATE improvements and build step fix

### PR DESCRIPTION
**Pull request by Jacalz**

### CODE REVIEW

**Comment by Jacalz** [comment]
Please wait with merging this, it is not ready yet 😉

**Comment by Jacalz** [comment]
This is ready for review now 🙂

**Comment by srirambv** [comment]
I'd suggest to use this for mobile repository. Follows same structure as laptop but customized for mobile repository

```
<!-- Have you searched for similar issues on the repository?
Before submitting this issue, please visit our wiki for common ones: https://github.com/brave/browser-laptop/wiki
For more, check out our community site: https://community.brave.com/ -->

### Description: 
<!--[Provide comprehensive description of the issue] -->

### Steps to Reproduce
<!-- Please add a series of steps to reproduce the problem.-->

  1.
  2.
  3.

**Actual result:**
<!-- Add screenshots if needed -->

**Expected result:**

**Reproduces how often:** [Easily reproduced/Intermittent issue]

**Brave Version:**
<!-- Provide full details Eg: v1.4.2(17.09.08.16) -->

**Device details:**
<!-- Model type and iOS version Eg: iPhone 6s+ (iOS 10.3.3)-->

**Is this an issue in the currently released version?**

**Website problems only:** 
- did you check with Brave Shields down? 
- did you check in Puffin/Dolphin (UIWebView-based browsers)? 
- did you check in Safari/Firefox (WkWebView-based browsers)? 


**Additional Information**
<!--Any additional information, related issues, extra QA steps, configuration or data that might be necessary to reproduce the issue.-->
```

cc: @kjozwiak @jhreis

**Comment by Jacalz** [comment]
This is updated as requested @srirambv

File: README.md [hunk 1]
```diff
@@ -28,6 +28,11 @@ brew install carthage
 git clone https://github.com/brave/browser-ios.git # this will download git if it isn't currently installed
 ```
 
+<strong> 6. Run the following command: </strong>
+```
+cd browser-iOS && (cd brave && ./setup.sh your.fake.appid) && open Client.xcodeproj
```
Comment (jhreis) [inline README.md]: This is super confusing. I understand it is nice to have in one line, but really unclear what is happening. I think we should break it out into 3 lines like it was before.

Comment (Jacalz) [reply README.md]: Fixed

**Pull request merged**